runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileContent/ProcessTileContent.tsx

```diff
@@ -1,66 +1,66 @@
-            </Box>
-            <Box>
-                <Typography color="textSecondary" variant="body2">
-                    {translate('Component.Tile.Process.Content.Creator')}
-                </Typography>
-                <Typography color="textPrimary" variant="h6">
-                    <HighlightText
-                        text={process.createdBy ? process.createdBy.login : 'RunBotics'}
-                        matchingText={searchValue}
-                    />
-                </Typography>
-            </Box>
-            <Box>
-                <Typography color="textSecondary" variant="body2">
-                    {translate('Component.Tile.Process.Content.Updated')}
-                </Typography>
-                <Typography color="textPrimary" variant="h6">
-                    {moment(process.updated).fromNow()}
-                </Typography>
-            </Box>
-            <Box>
-                <Typography color="textSecondary" variant="body2">
-                    {translate('Component.Tile.Process.Content.Scheduled')}
-                </Typography>
-                <If
-                    condition={process.schedules && process.schedules.length > 0}
-                    else={<RemoveCircleOutlineOutlinedIcon sx={{ color: red[500] }} />}
-                >
-                    <CheckCircleOutlineOutlinedIcon color="success" />
-                </If>
-            </Box>
+        <StyledContent ref={contentBoxRef}>
+            <StyledContainer>
+                <StyledBox>
+                    <Typography color='textPrimary' variant='h6'>
+                        {translate('Component.Tile.Process.Content.LastRun')}
+                    </Typography>
+                    <Typography color='textSecondary' variant='body2'>
+                        {process.lastRun
+                            ? moment(process.lastRun).fromNow()
+                            : translate('Component.Tile.Process.Content.LastRun.Placeholder')
+                        }
+                    </Typography>
+                </StyledBox>
+                <StyledBox>
+                    <Typography color='textPrimary' variant='h6'>
+                        {translate('Component.Tile.Process.Content.Updated')}
+                    </Typography>
+                    <Typography color='textSecondary' variant='body2'>
+                        {moment(process.updated).fromNow()}
+                    </Typography>
+                </StyledBox>
+            </StyledContainer>
+            <VerticalLine/>
+            <StyledContainer>
+                <StyledBox>
+                    <Typography color='textPrimary' variant='h6'>
+                        {translate('Component.Tile.Process.Content.Creator')}
+                    </Typography>
+                    <Typography color='textSecondary' variant='body2'>
+                        <HighlightText
+                            text={process.createdBy ? process.createdBy.login : 'RunBotics'}
```

Comment:
this 'RunBotics' could be a const AUTHOR_PLACEHOLDER

Reply:
or better let's make a translate('...............') like above

Reply:
Done.